### DISCUSSION TITLE
Using denyallow rule modifier with regex rule

### DISCUSSION THREAD

**Discussion by emlimap** [AdguardTeam/AdGuardHome#4115] [Q&A]
I am trying to use denyallow modifier with regex, but it doesn't seem to block DNS rebinding. 
```
/^192.168(.[0-9]{1,3}){2}$/$denyallow=home.arpa
```
It works fine without denyallow

```
/^192.168(.[0-9]{1,3}){2}$/
```

Test domain I used for this was `c0a80101.c0a80001.rbndr.us` which returns either 192.168.0.1 or 192.168.1.1 in random

Not sure if this is supported as documentation only list important flag https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#regular-expressions

**Comment by emlimap** [comment]
@ameshkov @ainar-g Friendly nudge. Any help on this would be appreciated.

**Reply by ameshkov** [reply]
@emlimap could you please explain what domains you are trying to block (unblock).

Are you trying to block all domains that point to IP addresses in the range `192.168.*`?
What domains would you like to exclude from this then?
